plaso/parsers/ios_mobile_installation_logs.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+"""Parser for iOS mobile installation logs files obtained from iOS device.
+Based on the research by Alex Brignoni.
```

Comment:
Please don't put links of blog posts in source, these go out of date very quickly. Instead please create a forensicwiki article.